packages/lua-ml/lua-ml.0.9/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "An embeddable Lua 2.5 interpreter implemented in OCaml"
+maintainer: "Daniil Baturin <daniil+opam@baturin.org>"
+authors: [
+  "Norman Ramsey <nr@cs.tufts.edu>" "Christian Lindig <lindig@gmail.com>"
+]
+license: "Two-clause BSD"
+homepage: "https://github.com/lindig/lua-ml"
+bug-reports: "https://github.com/lindig/lua-ml/issues"
+dev-repo: "git+https://github.com/lindig/lua-ml.git"
+depends: [
+  "ocaml" {>= "4.07"}
+  "ocamlbuild"
+  "ocamlfind"
```

Comment:
```suggestion
```
I haven't found any use of ocamlfind anywhere in the code, I think is an artefact of a use of a different build system